cygnus-ngsi-ld/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgreSQLSink.java

```diff
@@ -479,10 +479,11 @@ public void aggregate(NGSILDEvent cygnusEvent) {
                     for (Map.Entry<String, JsonElement> entry2 : y.entrySet()) {
                         String x2 = entry2.getKey();
                         Object y2 = entry2.getValue();
-                        if ("observedAt".contentEquals(x2)){
+                        if ("observedAt".contentEquals(x2) || "unitCode".contentEquals(x2)){
```

Comment:
A short sentence describing the fix done by this PR should be added to CHANGES_NEXT_RELEASE file.

Reply:
Indeed, added.